wallet/how-to/use-non-evm-networks/starknet/manage-starknet-networks.md

```diff
@@ -0,0 +1,41 @@
+---
+description: Manage Starknet networks in MetaMask.
+sidebar_position: 3
+---
+
+# Manage Starknet networks
+ 
+## Switch between networks
+
+Allow users to switch between different Starknet networks, such as mainnet and testnet. This function enables switching by setting the 
+appropriate chain Id. Starknet network has two official networks: 
```

Comment:
```suggestion
You can allow users to switch between different Starknet networks using the `wallet_switchStarknetChain` method and setting the 
appropriate chain ID. Starknet has two official networks: 
```

Where is `wallet_switchStarknetChain` documented? is this specific to `get-starknet`?